src/cassandra/inet.rs

```diff
@@ -17,11 +17,17 @@ use std::string::ToString;
 
 #[repr(C)]
 /// Cassandra's version of an IP address
-
+#[derive(Copy,Clone)]
 pub struct Inet(_Inet);
 
 impl Debug for Inet {
-    fn fmt(&self, f: &mut Formatter) -> fmt::Result { write!(f, "can't format an inet") }
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result { write!(f, "{}", self.to_string()) }
+}
+
+impl PartialEq for Inet {
+    fn eq(&self, other: &Inet) -> bool {
+        self.0.address == other.0.address
```

Comment:
This isn't strictly right - for IPv4 it will compare some irrelevant bytes.  You should compare `address_length`, and then if they're the same compare the first `address_length` bytes only.

Reply:
Very good catch, I spotted an issue with the `Inet` value building.